clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnection.java

```diff
@@ -179,10 +179,9 @@ public HubConnectionState getConnectionState() {
 
     /**
      * Starts a connection to the server.
-     *
-     * @throws Exception An error occurred while connecting.
+     * @return A completable future that represents starting.
```

Comment:
nit: `A completable future that completes when the connection has been established`